config/server-vars.yml

```diff
@@ -100,6 +100,7 @@ EDXAPP_FEATURES:
 EDXAPP_COMPREHENSIVE_THEME_DIRS: %%EDXAPP_COMPREHENSIVE_THEME_DIRS%%
 EDXAPP_ENABLE_COMPREHENSIVE_THEMING: %%EDXAPP_ENABLE_COMPREHENSIVE_THEMING%%
 EDXAPP_DEFAULT_SITE_THEME: "%%EDXAPP_DEFAULT_SITE_THEME%%"
+COMPREHENSIVE_THEME_DIRS: "{{ EDXAPP_COMPREHENSIVE_THEME_DIRS }}"
```

Comment:
server-vars.yml shouldn't have COMPREHENSIVE_THEME_DIRS. ANSIBLE will copy EDXAPP_COMPREHENSIVE_THEME_DIRS as COMPREHENSIVE_THEME_DIRS.
I think this is the reason we started seeing issues with theming. You should remove this line. Theming will be installed in oxa playbooks.